fuzzamoto-ir/src/lib.rs

```diff
@@ -59,6 +59,94 @@ pub struct FullProgramContext {
     pub txos: Vec<Txo>,
     /// List of headers present in the snapshotted state
     pub headers: Vec<Header>,
+    /// Addresses that can be relayed to peers
+    pub addresses: Vec<AddrRecord>,
```

Comment:
I think for now we should remove this from the context. This might make more sense in the future if we give nodes in the test non-local IPs and have them auto connect to each other.